app/process/handler/transfer/transfer.go

```diff
@@ -70,11 +66,12 @@ func NewHandler(
 		transactionRepo:    transactionRepository,
 		logger:             config.GetLoggerFor("Account Transfer Handler"),
 		feeCalculator:      feeCalculator,
+		contractService:    contractService,
 	}
 }
 
 // Recover mechanism
-func (cth Handler) Recover(q *queue.Queue) {
+func (cth *Handler) Recover(q *queue.Queue) {
```

Comment:
nit: no need for `*`. We do not set anything on the `Handler` struct so we do not need the `*`